### ISSUE TITLE
Add advanced settings to change default CrossOver bottle directory

### ISSUE THREAD

**Issue by peaz** [Heroic-Games-Launcher/HeroicGamesLauncher#3177]
### Problem description

CrossOver actually allows you to change the default bottle directory ( /Users/${USER}/Library/Application Support/CrossOver/Bottles) to any other location, usually an external drive volume. Doing so breaks Heroic's support for CrossOver as it would not be able to find the Heroic bottle.

https://www.codeweavers.com/support/wiki/mac/faq/cxofficemacbottlelocation.

### Feature description

There should also be a way to set/change the CrossOver bottle base directory from the default one.

**Comment by avv3lenato** [comment]
It would be a great feature. All my CrossOver bottles are on an external drive.

**Comment by Space-Cadet0** [comment]
Agreed, games are getting larger and mac hard drives certainly aren't increasing in size

**Comment by Snarj** [comment]
sorely needed feature.

**Comment by SpyrosKat** [comment]
How is this not implemented yet? Heroic is critical for Macs, and macs don't have expandable storage, external storage of games is inevitable. Please add this essential feature.

**Comment by arielj** [comment]
> How is this not implemented yet?

PRs are welcomed, this is open source software maintained on the free time of many developers
